Source/Krypton Components/Krypton.Workspace/Krypton.Workspace 2022.csproj

```diff
@@ -99,4 +99,17 @@
     <Optimize>True</Optimize>
   </PropertyGroup>
 
+	<!--Custom pre-processor directives-->
```

Comment:
Please use a single file, then use a shared reference into each of the other projects